cli/root.go

```diff
@@ -100,6 +100,7 @@ func (r *RootCmd) CoreSubcommands() []*serpent.Command {
 		r.publickey(),
 		r.resetPassword(),
 		r.state(),
+		r.task(),
```

Comment:
suggest putting this under `exp` for now

Reply:
I've also added the `exp task` subcommand in my PR #19496, perhaps we should add it in a separate PR since all of us are working on the same thing. 😄